scripts/limsquery.py

```diff
@@ -0,0 +1,24 @@
+
+from genologics.entities import *
+from genologics.lims import *
+from genologics.config import BASEURI, USERNAME, PASSWORD
+
+
+
+
+lims= Lims(BASEURI, USERNAME, PASSWORD)
+
+
+def  get_udfs():
+
+    step_name='End repair, A-tailing and adapter ligation (TruSeq RNA) 4.0'
```

Comment:
You're right, I could go all fancy and add options, make it work for any
amount of steps or udfs, etc...
But the whole point was to get something out quick so he could get the data
he was looking for. If you wish to implement it properly, be my guest ;)